.github/stale.yml

```diff
@@ -15,4 +15,7 @@ issues:
         - bug
     markComment: >
         This issue has been automatically marked as stale because it has not had
-        recent activity. It will be closed if no further activity occurs.
+        any activity in the last 2 years. If you feel that this is a valid issue,
+        just comment and the stale tag would be removed, otherwise it will be
+        closed in 7 days. This is an attempt to ensure that we have open issues that
```

Comment:
The end of this sentence is weird.  Maybe "This is an attempt to ensure that our open issues remain valuable and relevant so that we can keep track of what needs to be done and prioritize the right things".